contributrs-quick-start.rst

```diff
@@ -0,0 +1,976 @@
+
+   .. raw:: html
+
+      <div align="center" style="padding-bottom:10px">
+        <img src="images/quick_start/airflow_clone.png"
+             alt="Cloning github fork of Apache airflow">
+      </div>
+
+
+
+3. Open PyCharm and click ``Get from Version Control``
+
+   .. raw:: html
+
+      <div align="center" style="padding-bottom:10px">
+        <img src="images/quick_start/pycharm_clone.png"
+             alt="Cloning github fork to Pycharm">
+      </div>
+
+
+4. Paste the copied clone link in URL and click on clone.
+
+   .. raw:: html
+
+      <div align="center" style="padding-bottom:10px">
+        <img src="images/quick_start/click_on_clone.png"
+             alt="Cloning github fork to Pycharm">
+      </div>
+
+
+Setting up Breeze
+-----------------
+1. Open terminal and enter into virtual environment ``airflow-env`` and goto project directory
+
+.. code-block:: bash
+
+  $ pyenv activate airflow-env
+  $ cd ~/PycharmProjects/airflow/
+
+2. Initializing breeze autocomplete
+
+.. code-block:: bash
+
+  $ ./breeze setup-autocomplete
+  $ source ~/.bash_completion.d/breeze-complete
+
+3. Initialize breeze environment with required python version and backend. This may take a while for first time.
+
+.. code-block:: bash
+
+  $ ./breeze --python 3.8 --backend mysql
+
+4. Creating airflow tables and users. ``airflow db reset`` is required to execute at least once for Airflow Breeze to get
+   the database/tables created.
+
+.. code-block:: bash
+
+  $ airflow db reset
+  $ airflow users create --role Admin --username admin --password admin --email admin@example.com --firstname\
+    foo --lastname bar
+
+
+5. Closing Breeze environment. After successfully finishing above command will leave you in container,
+   type ``exit`` to exit the container
+
+.. code-block:: bash
+
+  root@b76fcb399bb6:/opt/airflow#
+  root@b76fcb399bb6:/opt/airflow# exit
+
+.. code-block:: bash
+
+  $ ./breeze stop
+
+6. Linking Local virtual environment ``airflow-env`` with breeze.
+
+   It may requires some packages to be installed.
+
+.. code-block:: bash
+
+  $ sudo apt-get install sqlite libsqlite3-dev default-libmysqlclient-dev postgresql
+  $ ./breeze initialize-local-virtualenv --python 3.8
+
+
+7. Add following line to ~/.bashrc in order to call breeze command from anywhere.
+
+.. code-block:: bash
+
+  export PATH=${PATH}:"/home/${USER}/PycharmProjects/airflow"
+  source ~/.bashrc
+
+Using Breeze
+-----------------------------------------
+
+1. Starting breeze environment using ``breeze start-airflow`` starts Breeze environment with last configuration run(
+   In this case python and backend will be picked up from last execution ``./breeze --python 3.8 --backend mysql``)
+   It also automatically starts webserver, backend and scheduler. It drops you in tmux with scheduler in bottom left
+   and webserver in bottom right. Use ``[Ctrl + B] and Arrow keys`` to navigate.
+
+.. code-block:: bash
+
+  $ breeze start-airflow
```

Comment:
You will have to update that slightly (including the screenshot). We are just about to merge a change #13313 where we have additional window started that watches and recompiles assets automatically for developing webserver. This is a cool addition and it would be great if you add information about it here :).

Reply:
Sure, i will update it when it's merged.

Reply:
Just merged! Please rebase and update and it will be ready to merge :).

Happy New Year!